Wbooks/Library/LibraryController.swift

```diff
@@ -41,11 +74,29 @@ final class LibraryController: UIViewController, UITableViewDelegate, UITableVie
                 
         let cell = tableView.dequeueReusableCell(withIdentifier: LibraryCell.identifier, for: indexPath) as! LibraryCell
         let book = booksArray[indexPath.row]
-        
+                
         cell.coverImage.image = book.cover
         cell.titleLabel.text = book.title
         cell.authorLabel.text = book.author
         
         return cell
     }
+    
+    func tableView(_ tableView: UITableView, didDeselectRowAt indexPath: IndexPath) {
```

Comment:
Instead of `didDeselectRowAt`, we should use `didSelectRowAt`. The former will pass as argument a row which has been deselected, while the latter will give you the one which has just been selected.